opflexagent/gbp_ovs_agent.py

```diff
@@ -312,18 +356,24 @@ def mapping_to_file(self, port, net_uuid, mapping, fixed_ips,
         if device_owner in [n_constants.DEVICE_OWNER_ROUTER_INTF]:
             return
         ips_ext = mapping.get('extra_ips') or []
-
+        mac = mapping.get('mac_address') or port.vif_mac
         mapping_dict = {
             "policy-space-name": mapping['ptg_tenant'],
             "endpoint-group-name": (mapping['app_profile_name'] + "|" +
                                     mapping['endpoint_group_name']),
             "interface-name": port.port_name,
-            "mac": mapping.get('mac_address') or port.vif_mac,
+            "mac": mac,
             "promiscuous-mode": mapping.get('promiscuous_mode') or False,
             "uuid": port.vif_id,
             'neutron-network': net_uuid}
 
-        ips = [x['ip_address'] for x in fixed_ips]
+        # Allocated IPs should be filtered by ownership.
+        if 'owned_addresses' in mapping:
```

Comment:
I  think owned_addresses is just the subset of allowed-addresses that is currently owned ("active"); regular fixed_ips are not included in it.